open-machine-learning-jupyter-book/assignments/ml-fundamentals/web-app.md

```diff
@@ -0,0 +1,15 @@
+# Web-app
```

Comment:
Why not use the original name? https://github.com/microsoft/ML-For-Beginners/blob/main/3-Web-App/1-Web-App/assignment.md

Name the title as `Try a different model`, while naming the file as `try-a-different-model.md`. Please follow this pattern for all.